test/razorpay/test_customer.rb

```diff
@@ -41,5 +41,24 @@ def test_fetch_specific_customer
       assert_instance_of Razorpay::Customer, customer
       assert_equal customer.id, @customer_id
     end
+
+    def test_payments_fetch_tokens
+      stub_get(%r{customers/#{@customer_id}/tokens$}, 'tokens_collection')
+      customers = Razorpay::Customer.fetch(@customer_id).fetchTokens
+      assert_instance_of Razorpay::Collection, customers, 'Customer not an instance of Razorpay::Customer class'
+      refute_empty customers.items, 'customers should be more than one'
```

Comment:
nit: error msg, these are not customers.

Reply:
changed that message and variable